UltimateTeam.Toolkit/Requests/LoginRequest.cs

```diff
@@ -162,7 +162,7 @@ private static string GetNucleusPersonaPlatform(Platform platform)
         private async Task<UserAccounts> GetUserAccountsAsync(Platform platform)
         {
             HttpClient.RemoveRequestHeader(NonStandardHttpHeaders.Route);
-            var route = $"https://utas.{(platform == Platform.Xbox360 || platform == Platform.XboxOne ? "external.s3" : "s2")}.fut.ea.com:443";
+            var route = $"https://utas.external.{(platform == Platform.Xbox360 || platform == Platform.XboxOne ? "external.s3" : "s2")}.fut.ea.com:443";
```

Comment:
Looking at https://github.com/trydis/FIFA-Ultimate-Team-Toolkit/issues/316#issuecomment-270002854 it looks like my guess was correct. 

Are you able to push another another commit which changes `external.s3` to `s3`? Just push to to your source branch and it will show up here. If not i'll just acccept this and do it afterwards.